src/pages/waiting-room-to-car/cat-c/components/vehicle-checks-modal/vehicle-checks-modal.cat-c.page.html

```diff
@@ -0,0 +1,62 @@
+<ion-header>
+  <ion-navbar>
+    <ion-buttons ion-buttons start navPop>
+      <button ion-button center class="nav-close" id="closeVehicleChecksModalCatBE">
```

Comment:
id references `CatBE`

Reply:
This Id is not used anywhere else. But I'm updating it to use `CatC` for now